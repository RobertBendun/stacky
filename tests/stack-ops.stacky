# dot compare
2 1 drop .
1 2 over . . .
3 dup . .
1 2 2dup . . . .
5 6 7 rot . . .
10 11 tuck . . .
20 21 22 2drop .
30 31 32 33 2swap . . . .
40 41 42 43 2over . . . . . .
