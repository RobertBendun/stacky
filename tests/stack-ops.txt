2
1
2
1
3
3
2
1
2
1
5
7
6
11
10
11
