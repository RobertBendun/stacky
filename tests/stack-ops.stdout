2
1
2
1
3
3
2
1
2
1
5
7
6
11
10
11
20
31
30
33
32
41
40
43
42
41
40
